addons/l10n_ro_edi_etransport/models/l10n_ro_edi_etransport_document.py

```diff
@@ -0,0 +1,563 @@
+import markupsafe
+import requests
+import re
+
+from odoo import api, fields, models, _
+
+DOCUMENT_STATES = [
+    ('draft', "Draft"),
+    ('sending', "Sending"),
+    ('sending_failed', "Error"),
+    ('amending', "Sending"),
+    ('amending_failed', "Error"),
+    ('sent', 'Sent'),
```

Comment:
We could combine the sending(_failed) and amending(_failed) states, but then we could run into the following problem:
when the document is in the `sending` state there is no way of knowing if we are amending to a document or sending a document for the first time. we do want to know this because we have different buttons for this. the solution would be to have a field hold the send type which is IMO less practical and less clear

As for prefixing the states: if we prefix them it should be done with `document_` and not `picking_` because the document is completely separate from pickings. and prefixing with document sounds a bit redundant to me